alacritty/src/display/mod.rs

```diff
@@ -545,16 +545,20 @@ impl Display {
 
     fn swap_buffers(&self) {
         #[allow(clippy::single_match)]
-        match (self.surface.deref(), &self.context.get()) {
+        let res = match (self.surface.deref(), &self.context.get()) {
             #[cfg(not(any(target_os = "macos", windows)))]
             (Surface::Egl(surface), PossiblyCurrentContext::Egl(context))
                 if self.is_wayland && !self.debug_damage =>
             {
                 surface.swap_buffers_with_damage(context, &self.damage_rects)
             },
             (surface, context) => surface.swap_buffers(context),
+        };
+        // We ignore the errors, but log them into `debug` so we can at least
+        // troubleshoot them in case they could be suspicious to real issues.
```

Comment:
```suggestion

```

Was thinking how to improve this doc message since it sounds a bit weird, but honestly developers can probably read and we don't need to explicitly document this.

Reply:
I don't how to improve it either, I just stated that we log here for our own purposes, and it's not really a user warning or anything like that. They don't show real issues from what we know and only a subject to potential issues.

Reply:
I mean that's exactly what `debug!` says anyway though. Debug logs are stuff we log for troubleshooting that aren't user errors.

Reply:
Hm, I guess, so.